appstore-metadata-service/src/main/java/com/lgi/appstore/metadata/api/converter/StringToApplicationTypeConverter.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2022 Liberty Global Technology Services BV
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.lgi.appstore.metadata.api.converter;
+
+import com.lgi.appstore.metadata.model.ApplicationType;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import static java.lang.String.format;
+import static java.util.Optional.ofNullable;
+
+@Component
+public class StringToApplicationTypeConverter implements Converter<String, ApplicationType> {
+
+    @Override
+    public ApplicationType convert(String name) {
+        try {
+            return ApplicationType.valueOf(ofNullable(name).orElse(""));
```

Comment:
It's not clear what type of object is created here when static import is introduced. I like static imports, but for such factory method I would using avoid it